js/boaexchange.js

```diff
@@ -0,0 +1,1163 @@
+                        'ledger',
+                        'markets',
+                        'markets/{label}',
+                        'markets/{label}/fees',
+                        'markets/{label}/limits',
+                        'markets/{label}/ohlcv',
+                        'markets/{label}/orderbook',
+                        'news',
+                        'orders',
+                        'orders/{orderId}',
+                        'trades',
+                        'trades/{tradeId}',
+                        'trades/all',
+                        'transactions',
+                        'transfers',
+                        'transfers/{transferId}',
+                        'withdraws',
+                    ],
+                    'post': [
+                        'addresses/{label}',
+                        'airdrops/{airdropId}',
+                        'apiKeys',
+                        'apiKeys/{key}',
+                        'chat',
+                        'markets/favorites',
+                        'markets/{label}/toggle',
+                        'orders',
+                        'transfers',
+                        'withdraws',
+                    ],
+                    'delete': [
+                        'apiKeys',
+                        'orders',
+                        'orders/{orderId}',
+                        'withdraws/{withdrawId}',
+                    ],
+                },
+            },
+            'requiredCredentials': {
+                'apiKey': true,
+                'secret': true,
+            },
+            'dontGetUsedBalanceFromStaleCache': true,
+            'fees': {
+                'trading': {
+                    'tierBased': false,
+                    'percentage': true,
+                    'maker': 0.15,
+                    'taker': 0.15,
+                },
+                'funding': {
+                    'tierBased': false,
+                    'percentage': false,
+                    'withdraw': {
+                    },
+                    'deposit': {
+                    },
+                },
+            },
+            'exceptions': {
+                'Invalid X-BOA-ENCRYPTED': AuthenticationError,
+                'Invalid X-BOA-NONCE': InvalidNonce,
+                'Address is Pending': AddressPending,
+                'Insufficient Funds': InsufficientFunds,
+                'Order Not Found': OrderNotFound,
+                'Too Many Requests': DDoSProtection,
+            },
+            'options': {
+                'hasEmptyMarkets': true,
+                // price precision by quote currency code
+                'pricePrecisionByCode': {
+                    'USD': 3,
+                },
+                'symbolSeparator': '_',
+                'tag': {
+                },
+            },
+            'commonCurrencies': {
+            },
+        });
+    }
+
+    costToPrecision (symbol, cost) {
```

Comment:
If they use DECIMAL_PLACE then we can remove it from here, because it's in the base class.

Reply:
done.